includes/class-cvc-query.php

```diff
@@ -77,16 +77,19 @@ public function alter_query_parameters( $args ) {
 		}
 		// hidden filters
 		$fields = $this->cvc->api->call_values( 'DataProcessorFilter', 'get', [
-			'sequential'        => 1,
-			'is_required'       => 1,
 			'is_exposed'        => 1,
 			'data_processor_id' => $id
 		] );
 		foreach ( $fields as $field ) {
-			if ( $this->cvc->dp_options->has_option( $field['name'], Content_Views_CiviCRM_Dp_Option::USER_CONTACT_ID ) ) {
+			if ( $field['is_required'] && $this->cvc->dp_options->has_option( $field['name'], Content_Views_CiviCRM_Dp_Option::USER_CONTACT_ID ) ) {
```

Comment:
Only get the logged in user when the field is required? Sounds good to me.

Reply:
:+1: